modules/payment/commerce_payment.module

```diff
@@ -125,12 +139,16 @@ function commerce_payment_create_field_definitions(array $definitions) {
  *
  * @param array $definitions
  *   The field definitions.
+ * @param bool $delete_storage
+ *   If set TRUE, field storage definition will be deleted also
```

Comment:
But does it delete the payment entities? What are the ramifications for viewing past orders and payment entities?